### PR TITLE
Dashboard deploy to cloud michael

### PR DESCRIPTION
**Pull request by MichaelHassan**
Adds 
- Docker file 
- Terraform to deploy dashboard.

### CODE REVIEW

File: dashboard/Dashboard.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 # Download the 'punkt' tokenizer model
 nltk.download('punkt_tab')
+nltk.download('stopwords')
```
Comment (paulauus) [inline dashboard/Dashboard.py:18]: Now we know.

**Review by paulauus — DISMISSED** [review]
Thank you, terraform wizard.

**Review by Leon-JS02 — DISMISSED** [review]
Tested - dashboard has been deployed successfully. Thanks for handling this.

**Review by Leon-JS02 — APPROVED** [review]
This was a huge bug, good job spotting that problem.

**Review by mehrdadhalali — APPROVED** [review]

**Review by paulauus — APPROVED** [review]

**Pull request merged**